x-pack/plugins/apm/public/components/app/RumDashboard/UserPercentile/index.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useCallback, useEffect } from 'react';
+
+import { EuiSuperSelect } from '@elastic/eui';
+import { useHistory } from 'react-router-dom';
+import { useUrlParams } from '../../../../hooks/useUrlParams';
+import { fromQuery, toQuery } from '../../../shared/Links/url_helpers';
+
+const DEFAULT_P = 50;
+
+export function UserPercentile() {
+  const history = useHistory();
+
+  const {
+    urlParams: { percentile },
+  } = useUrlParams();
+
+  const updatePercentile = useCallback(
+    (percentileN?: number) => {
+      const newLocation = {
+        ...history.location,
+        search: fromQuery({
+          ...toQuery(history.location.search),
+          percentile: percentileN,
+        }),
+      };
+      history.push(newLocation);
+    },
+    [history]
+  );
+
+  useEffect(() => {
+    if (!percentile) {
+      updatePercentile(DEFAULT_P);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const options = [
+    {
+      value: '50',
+      inputDisplay: 'P50 (Median)',
+      'data-test-subj': 'p50Percentile',
+    },
+    {
+      value: '75',
+      inputDisplay: 'P75',
+      'data-test-subj': 'p75Percentile',
+    },
+    {
+      value: '90',
+      inputDisplay: 'P90',
+      'data-test-subj': 'p90Percentile',
+    },
+    {
+      value: '95',
+      inputDisplay: 'P95',
+      'data-test-subj': 'p95Percentile',
+    },
+    {
+      value: '99',
+      inputDisplay: 'P99',
+      'data-test-subj': 'p99Percentile',
+    },
+  ];
```

Comment:
Can I make a suggestion to rename the percentiles to "50th (median), 75th, 90th" etc. possibly with the added title before the options labelled "Percentiles"? The reason being I don't think the average user will know that P50 etc. refers to. cc @drewpost cc @katrin-freihofner

Reply:
I like this usability suggestion. Thanks @formgeist I agree - anything we can do to help people understand this better, we should do.